src/bin/cmjava.rs

```diff
@@ -104,9 +104,8 @@ fn main() -> anyhow::Result<()> {
     log_setup(cli.verbose);
 
     let mut heap = Heap::default();
-    let class_files = cli.class_file.unwrap();
     let mut bytecode_classes: Vec<Rc<dyn Class>> = Vec::new();
-    for class_file in class_files {
+    for class_file in cli.class_file {
```

Comment:
This reads a little suprising due to the singular name of the variable. Have you considered renaming it?

Reply:
we renamed this to class_files, this should be fixed